src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+                evalSkippedTesting.add( new VariantContextBuilder(vc).attribute(VARIANT_SKIPPED_EVAL_REASON_KEY, "in_masked_out_region").make() );
+                return false;
+            }
+
+            if ( minMQ > 0 ) {
+                if (vc.hasAttribute(GATKSVVCFConstants.MAPPING_QUALITIES)) {
+                    int maxMQ = SVUtils.getAttributeAsStringStream(vc, GATKSVVCFConstants.MAPPING_QUALITIES).mapToInt(Integer::new).max().orElse(0);
+                    if (maxMQ < minMQ) {
+                        evalSkippedTesting.add( new VariantContextBuilder(vc).attribute(VARIANT_SKIPPED_EVAL_REASON_KEY, "low_asm_mq").make() );
+                        return false;
+                    }
+                }
+            }
+            if ( minAlnLen > 0 ) {
+                if (vc.hasAttribute(GATKSVVCFConstants.MAX_ALIGN_LENGTH)) {
+                    if (vc.getAttributeAsInt(GATKSVVCFConstants.MAX_ALIGN_LENGTH, 0) < minAlnLen) {
+                        evalSkippedTesting.add( new VariantContextBuilder(vc).attribute(VARIANT_SKIPPED_EVAL_REASON_KEY, "short_asm_aln").make() );
+                        return false;
+                    }
+                }
+            }
+
+            final int svLen = Math.abs(vc.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0));
+            final String svType = vc.getAttributeAsString(GATKSVVCFConstants.SVTYPE, "");
+            if (svType.equals(SimpleSVType.SupportedType.DEL.name())) { // some tiny deletion calls are actually RPL calls with micro deletions
+                if ( (!vc.hasAttribute(GATKSVVCFConstants.INSERTED_SEQUENCE)) && svLen < StructuralVariationDiscoveryArgumentCollection.STRUCTURAL_VARIANT_SIZE_LOWER_BOUND) { // no ins_seq AND tiny
+                    evalSkippedTesting.add( new VariantContextBuilder(vc).attribute(VARIANT_SKIPPED_EVAL_REASON_KEY, "tiny_del").make() );
+                    return false;
+                }
+            } else if (svType.equals(SimpleSVType.SupportedType.INS.name()) || svType.equals(SimpleSVType.SupportedType.DUP.name())) {
+                if (svLen < StructuralVariationDiscoveryArgumentCollection.STRUCTURAL_VARIANT_SIZE_LOWER_BOUND) {
+                    evalSkippedTesting.add( new VariantContextBuilder(vc).attribute(VARIANT_SKIPPED_EVAL_REASON_KEY, "tiny_ins").make() );
+                    return false;
+                }
+            }
+
+            return true;
+        };
+    }
+
+    /**
+     * A 2-step evaluation mechanism:
+     * <ul>
+     *     <li> first test if {@code eval} (padded by {@link #getPaddedSvInterval(VariantContext)}) overlaps with any record in the truth tree </li>
+     *     <li> further test concordance if there's such overlap </li>
+     * </ul>
+     */
+    @Override
+    protected void apply(final VariantContext eval, final ReadsContext readsContext, final ReferenceContext refContext) {
+        final SVInterval evalInterval = getPaddedSvInterval(eval);
```

Comment:
ReadsContext is never used and since this is the only extending class for the abstract parent I would remove it from the signature.

Reply:
overriding `VariantWalker` now, so cannot change it.